backend/app/requirements.txt

```diff
@@ -1,21 +1,33 @@
+alembic==0.9.9
+cairocffi==0.8.0
+CairoSVG==2.0.3
+cffi==1.11.5
 click==6.7
-flask==0.12.2
+defusedxml==0.5.0
+Flask==0.12.2
+Flask-Compress==1.4.0
+Flask-Cors==3.0.3
+Flask-Migrate==2.1.1
+Flask-SQLAlchemy==2.1
+gunicorn==19.7.1
 itsdangerous==0.24
-jinja2==2.10
-markupsafe==1.0
-werkzeug==0.14.1
-wtforms==2.1
-pyflakes==1.6.0
-flask-compress==1.4.0
-selenium==3.11.0
+Jinja2==2.10
+jsonschema==2.6.0
+lxml==4.2.1
+Mako==1.0.7
 nose==1.3.7
-pypdf2==1.26.0
-defusedxml==0.5.0
-cairocffi==0.8.0
-cairosvg==2.0.3
-flask-cors==3.0.3
-psycopg2==2.7.4
+Pillow==5.1.0
+psycopg2-binary==2.7.4
+pycparser==2.18
+pyflakes==1.6.0
 PyJWT==1.4.2
-Flask-SQLAlchemy==2.1
-Flask-Mail==0.9.1
-
+PyPDF2==1.26.0
+python-dateutil==2.7.2
```

Comment:
I think It is not used in our project.

Reply:
I haven't added anything, it's sub-dependencies. I just did `pip freeze > requirements.txt` to add the installed libraries